index.js

```diff
@@ -0,0 +1,102 @@
+import { mobilePopupData } from './data.js';
+
+const navtext = document.querySelector('.mobile-menu');
+const hamburger = document.querySelector('.menubar');
+const closeBtn = document.querySelector('.closebtn');
+const menuLists = document.querySelectorAll('.menu-list');
+
+hamburger.addEventListener('click', () => {
+  navtext.style.display = 'block';
+});
+
+closeBtn.addEventListener('click', () => {
+  navtext.style.display = 'none';
+});
+
+for (let i = 0; i < menuLists.length; i += 1) {
+  menuLists[i].addEventListener('click', () => {
+    navtext.style.display = 'none';
+  });
+}
+
+document.getElementById('seeProjectButton').addEventListener('click', (event) => {
+  event.preventDefault();
+  // const button = event.target;
+  const mobilePopup = document.getElementById('mobile-popup');
+  // const buttonRect = button.getBoundingClientRect();
+  // const buttonCenterX = buttonRect.left + buttonRect.width + 25;
+  // mobilePopup.style.left = `${buttonCenterX}px`;
+  // mobilePopup.style.top = `${buttonRect.top - mobilePopup.offsetHeight}px`;
```

Comment:
- [ ] You have done great job :clap: However, Please do not commit old pieces of code as inline comments. Leaving old pieces of code as comments can clutter the codebase and make it harder to read and understand. Over time, as more code comments accumulate, it becomes challenging to differentiate between actual code and commented-out sections. This clutter can hinder the development process and introduce confusion for developers trying to navigate and maintain the code. :slightly_smiling_face:

Reply:
Hy @Alusp ,please some issues here are not clear to me.